tests/test-cases/core.yaml

```diff
@@ -44,7 +44,7 @@ tests:
       stdout:
         # build/atmos is local
         # atmos/atmos is in GitHub Actions
-        - '(build[/\\]atmos|atmos/atmos)'
+        - "/atmos"
```

Comment:
No, not good. This would match `/usr/bin/atmos`, and that's the thing we want to avoid here. We are checking that the version of atmos used, is the one that was compiled.